emcc.py

```diff
@@ -1577,8 +1577,11 @@ def get_final():
 
     with ToolchainProfiler.profile_block('source transforms'):
       # Embed and preload files
-      if shared.Settings.SPLIT_MEMORY:
-        options.no_heap_copy = True # copying into the heap is risky when split - the chunks might be too small for the file package!
+      # copying into the heap is risky when split - the chunks might be too small for the file package!
+      # Also, MEMFS is not aware of heap resizing feature in wasm, so if MEMFS and Wasm memory growth are used together, force
+      # no_heap_copy to be enabled.
+      if shared.Settings.SPLIT_MEMORY or (shared.Settings.BINARYEN and shared.Settings.ALLOW_MEMORY_GROWTH):
```

Comment:
isn't this an issue with asm.js memory growth too, not just wasm?

Reply:
True